sdk/highlight-react/build.js

```diff
@@ -0,0 +1,24 @@
+const esbuild = require('esbuild')
+const cssModulesPlugin = require('esbuild-css-modules-plugin')
+const isWatch = process.argv.includes('--watch')
+const pkg = require('./package.json')
+const config = {
+	logLevel: 'info',
+	entryPoints: ['src/index.tsx'],
+	bundle: true,
+	plugins: [cssModulesPlugin()],
```

Comment:
I needed the `cssModulesPlugin` and plugins can only be used with esbuild's `node` api... so bye-bye `tsup`, hello `build.js`